config/api.yml

```diff
@@ -1193,6 +1193,7 @@
     :description: Settings
     :identifier: ops_settings
     :options:
+    - :arbitrary_resource_path
     - :collection
```

Comment:
Is it meaningful for this to still be a collection *and* an arbitrary_resource_path?